### ISSUE TITLE
[FEATURE] Sprint Alldirections option for scaffold

### ISSUE THREAD

**Issue by Hartzender** [CCBlueX/LiquidBounce1.8-Issues#3342]
add an opition in scaffold for alldirection sprint to be turned on, for example for hive you need alldirection sprint for sprint scaffold but it flags you when you use alldirection normaly, only works with scaffold :/

**Comment by Soulplexis** [comment]
it's supposed to hook into sprint module, checkserverside etc.

**Comment by Hartzender** [comment]
yeah i just want an option to only have it with the scaffold module u know so u dont have alldirection sprint when u walk normaly.
